web/concrete/controllers/single_page/dashboard/system/seo/urls.php

```diff
@@ -1,144 +1,173 @@
 <?php
 namespace Concrete\Controller\SinglePage\Dashboard\System\Seo;
 
-use \Concrete\Core\Page\Controller\DashboardPageController;
-use Loader;
-use Config;
+use Concrete\Core\Page\Controller\DashboardPageController;
+
 class Urls extends DashboardPageController
 {
     /**
-    * Returns the mod_rewrite rules
-    *
-    * @return string
-    */
-    public function getRewriteRules()
-    {
-        $strRules = '
-<IfModule mod_rewrite.c>
-RewriteEngine On
-RewriteBase ' . DIR_REL . '/
-RewriteCond %{REQUEST_FILENAME} !-f
-RewriteCond %{REQUEST_FILENAME}/index.html !-f
-RewriteCond %{REQUEST_FILENAME}/index.php !-f
-RewriteRule . ' . DISPATCHER_FILENAME .' [L]
-</IfModule>';
-
-        return $strRules;
-    }
-
-    /**
-    * Returns the .htaccess text to be copied/inserted
-    *
-    * @return string
-    */
-    public function getHtaccessText()
+     * Dashboard page view.
+     *
+     * @param string|bool $strStatus - Result of attempting to update rewrite rules
+     * @param bool $prettyUrlState - Flag denoting if the rewrite rule has been saved
+     */
+    public function view($strStatus = false, $prettyUrlState = '')
```

Comment:
I agree. I just kept the old variable name... I'm somehow attached to it: it has been there for almost 5 years... (see https://github.com/concrete5/concrete5/commit/ebbf1e839f92d2f369272090ae4093f9f6b67c01#diff-39f73d6047296f4ca3190da491e5199aR48 )